spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-functional-binder-sample/sink/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+spring.cloud.stream.bindings.convertUserMessageToString-in-0.destination=[topic]
+
+spring.cloud.function.definition=convertUserMessageToString
```

Comment:
If this is the only function in your AC you don't need this property, but it helps readability

Reply:
Funny story: because we have that reactive scheduler bean hanging around as part of our pubsub infrastructure, SCS assumes that there are at least two eligible functions. 
I'll probably submit a feature request for that exclusion annotation (a property would have to be set by the user application, so an annotation may work better in this scenario).

Reply:
Right, i could predict your response given the history ;)